app/templates/package.json

```diff
@@ -2,68 +2,69 @@
   "name": "@sample-app-name",
   "version": "0.0.0",
   "dependencies": {
-    "body-parser": "^1.3.0",
+    "body-parser": "^1.14.0",
     "cookie-parser": "^1.1.0",
     "express": "^4.4.1",
     "express-session": "^1.2.1",
     "morgan": "^1.6.0"
   },
   "devDependencies": {
-    "browser-sync": "^2.7.12",
+    "browser-sync": "^2.9.3",
     "chai": "^3.0.0",
-    "del": "^1.2.0",
-    "glob": "^5.0.10",
-    "gulp": "3.8.x",
+    "del": "^2.0.2",
+    "glob": "^7.0.0",
+    "gulp": "^3.8.0",
     "gulp-angular-templatecache": "^1.6.0",
-    "gulp-autoprefixer": "^2.3.1",
+    "gulp-autoprefixer": "^2.2.0",
```

Comment:
Note: gulp-autoprefixer@2.3.1 does not exist. 2.2.0 is latest minor. And ^3.0.0 causes gulp to choke on nodejs 0.10 which is what is still used a lot..